### PR TITLE
#406 find_num_of_digits_in_int.cpp hakincodes

### PR DESCRIPTION
**Pull request by sukritidawar**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added the solution in cp folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by sukritidawar** [comment]
@HarshCasper @Kajol-Kumari please review.

File: C-Plus-Plus/cp/find_no_of_digits_in_int.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    
+    return 1 + num_of_digits_recursive(num / 10);        
+}
+
+int main() {
+    int num;
+    cin >> num;                                           //Inputting the number
+    cout << "The number of digits in given integer is ";
+    //cout << num_of_digits(num) << endl;                 //Normal approach
```
Comment (Kajol-Kumari) [inline C-Plus-Plus/cp/find_no_of_digits_in_int.cpp]: @sukritidawar Please uncomment this line and add two `cout` statements for each of normal and recursive approach.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please incorporate the suggested changes, Rest LGTM :)

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :) @sukritidawar please resolve the conflicts and it will be good to go!

**Comment by sukritidawar** [comment]
> LGTM :) @sukritidawar please resolve the conflicts and it will be good to go!

Done!

**Review by vatsalkesarwani12 — APPROVED** [review]
code looks good to me

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Code is good, I want you to add two more methods which can reduce the lines of code as well,

1 - Convert the number to a string, count the length of string using C++ functionality, two lines of code. While coding in any language try to use the language-specific desserts as well because that's important.

2 - Try something using Step function and Log Function. Hint - what happens if we add step function to the logarithm of a number with base 10? Is there any relation possible?

Do work on it and let me know.

**Comment by sukritidawar** [comment]
> Code is good, I want you to add two more methods which can reduce the lines of code as well,
> 
> 1 - Convert the number to a string, count the length of string using C++ functionality, two lines of code. While coding in any language try to use the language-specific desserts as well because that's important.
> 
> 2 - Try something using Step function and Log Function. Hint - what happens if we add step function to the logarithm of a number with base 10? Is there any relation possible?
> 
> Do work on it and let me know.

Done

**Comment by iamrajiv** [comment]
@sukritidawar, please resolve merge conflicts.

**Comment by sukritidawar** [comment]
> @sukritidawar, please resolve merge conflicts.

resolved

**Comment by iamrajiv** [comment]
@sukritidawar LGTM!

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @sukritidawar for code

**Pull request merged**